deploy/platform/kubernetes/templates/feature-rabbitmq-monitor/opentelemetry-config.yaml

```
@@ -29,7 +29,7 @@
       regex: rabbitmq;metrics
     - source_labels: [ ]
       target_label: cluster
-      replacement: rabbitmq-cluster
+      replacement: demo-rabbitmq.{{ .Release.Namespace }}
```

```suggestion
      replacement: {{ include "common.names.fullname" .Subcharts.rabbitmq }}.{{ .Release.Namespace }}
```
rabbitMQ is using Helm Chart dependencies, you should use the same variable as the Chart's svc name rather than hard code. Please test it.